lib/datadog/core/telemetry/event.rb

```diff
@@ -113,7 +113,10 @@ def configuration
               conf_value('tracing.opentelemetry.enabled', !defined?(Datadog::OpenTelemetry::LOADED).nil?),
             )
             list << conf_value('logger.instance', config.logger.instance.class.to_s) if config.logger.instance
-            list << conf_value('appsec.enabled', config.dig('appsec', 'enabled')) if config.respond_to?('appsec')
+            if config.respond_to?('appsec')
+              list << conf_value('appsec.enabled', config.dig('appsec', 'enabled'))
+              list << conf_value('appsec.sca_enabled', config.dig('appsec', 'sca_enabled'))
```

Comment:
Looking at the other options (lines 110 and 113 in this file), the pattern appears to be `component.function.enabled`, i.e. in this case it would be `appsec.sca.enabled`.  Is the `x_enabled` pattern used elsewhere?